### ISSUE TITLE
Update README

### ISSUE THREAD

**Issue by szapp** [szapp/g2freeAim#102]
The README barely touches on how free aiming is actually implemented. This information should be added to make the core functionality more accessible.

**Comment by szapp** [comment]
This needs to be done before restructure can be merged into master (see #116).

**Comment by szapp** [comment]
Completely rework the README from scratch.

**Comment by szapp** [comment]
Also, give a clear overview of the functions and how they interact and depend on each other. Preferably using a simple and intuitive diagram.

**Comment by szapp** [comment]
It would be a good idea, to add a separate README into the `_intern/` directory to free up the root README and give information on how the internal stuff works.
Another README for the config in the `config/` directory would also be a good idea, leaving the root README for installation instructions and general information.
